parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -202,6 +205,14 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 			}
 		}
 
+		// Allow to change dedicated storage items (called by governance-like)
+		match call {
+			RuntimeCall::System(frame_system::Call::set_storage { items })
+				if items.iter().any(|(k, _)| k.eq(&bridging::XcmBridgeHubRouterByteFee::key())) =>
+				return true,
+			_ => (),
+		};
```

Comment:
Yep, that has caught my attention too. But maybe that is alleviated by the fact that only root (relay chain governance) may call this fn. So it works more like a safety filter, not a strict filter. But yep - we may change the comparison to do a strict cmp with `vec![XcmBridgeHubRouterByteFee::key()]` instead of `.iter().any()`

We may add a dedicated call to the `pallet-xcm-bridge-hub-router` and store it there. But it conflicts a bit with configuration options - like right now we're getting both the `BaseFee` and `ByteFee` from the pallet configuration. So having a dedicated calls for `ByteFee` and not having for the `BaseFee` seems not ok. But we can revert my suggestion (this: https://github.com/paritytech/parity-bridges-common/pull/2313#discussion_r1281417787) and store + manage both `BaseFee` + `ByteFee` by the pallet. It has some drawbacks, but I guess we'll be changing the configuration in the v2 anyway.

Could do both options

Reply:
Ok, pushed the quick fix: filter exclusively matches `XcmBridgeHubRouterByteFee::key`.

On `Config::(Base/Byte)Fee` vs `<pallet-storage>::(Base/Byte)Fee`, I guess it depends on whether we expect this to change often-ish. If we expect to set it once and have it forever, then config is better. If we expect periodic changes to it, might be better UX to have `setter` call.